hdr/sqlite_modern_cpp.h

```diff
@@ -460,6 +460,10 @@ namespace sqlite {
 			return sqlite3_last_insert_rowid(_db.get());
 		}
 
+		sqlite3_int64 rows_modified() const {
```

Comment:
I think this should return the same type as `sqlite3_changes`, so `int` instead of `sqlite3_int64`.

Reply:
Oh that's right, It should be a int. Thank you. By the way which IDE do you use to the project ?

Reply:
Personally I'm using [(Neo)vim](https://neovim.io/), so I don't use any IDE.
The other guys might use something different, AFAICT there is no standard environment for the project.

Reply:
Nice, I'm used to use netbeans on cross compile development. on local machine I'm using atom editor. I used atom to edit the project , but I don't know why the identation and whitespaces  inconsistencies happened.